astropy/units/structured.py

```diff
@@ -3,8 +3,6 @@
 This module defines structured units and quantities.
 """
 
-from __future__ import annotations  # For python < 3.10
```

Comment:
If `from __future__ import annotations` was only needed to enable writing down union types using the `|`-operator then the import can be safely removed now. However, in files that have an `if TYPE_CHECKING:` block removing these lines would require rewriting (possibly many) annotations as string literals, which doesn't sound helpful. Luckily it is simple to find out if this import is needed or not: if it is then removing it causes the code to fail already at import time.